### PR TITLE
refactor tests

### PR DESCRIPTION
**Pull request by AviAvni**

### CODE REVIEW

File: tests/flow/test_concurrent_query.py [hunk 1]
```diff
@@ -12,45 +12,27 @@ def query_aggregate(query):
...
-
-    actual_result = graph.query(query)
-    person_count = actual_result.result_set[0][0]
-    if person_count != len(people):
-        return False
-    
-    return True
+    return graph.query(query)
 
 def query_neighbors(query):
```
Comment (swilly22) [inline tests/flow/test_concurrent_query.py:17]: What's the difference between:
`query_aggregate `, `query_neighbors `, `query_write ` and `thread_run_query`
seems just need `thread_run_query`

**Review by swilly22 — CHANGES_REQUESTED** [review]

File: tests/flow/test_concurrent_query.py [hunk 2]
```diff
@@ -8,31 +8,16 @@
...
-    return graph.query(query)
-
 def thread_run_query(query):
     env = Env(decodeResponses=True)
     conn = env.getConnection()
     graph = Graph(GRAPH_ID, conn)
 
     try:
-        return graph.query(query)
+        result = graph.query(query)
```
Comment (swilly22) [inline tests/flow/test_concurrent_query.py:17]: Would it be possible to place a `barrier` between all processes in the same pool just before executing the query?

File: tests/flow/test_concurrent_query.py [hunk 3]
```diff
@@ -127,43 +112,70 @@ def test_05_concurrent_read_delete(self):
...
+        # invoke queries
+        m = pool.amap(thread_run_query, queries)
+
+        self.conn.flushall()
+
+        # wait for processes to return
+        m.wait()
+
+        # get the results
+        results = m.get()
```
Comment (swilly22) [inline tests/flow/test_concurrent_query.py:151]: do we want to validate `results ` ?

Comment (AviAvni) [reply tests/flow/test_concurrent_query.py:151]: sorry don't need run_concurrent here so there is validation

File: tests/flow/test_concurrent_query.py (hunk 2 above)
Comment (AviAvni) [reply tests/flow/test_concurrent_query.py:22]: done

**Review by swilly22 — APPROVED** [review]

**Pull request merged**